storybook/pages/TransferOwnershipAlertPopupPage.qml

```diff
@@ -0,0 +1,136 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Storybook 1.0
+import Models 1.0
+
+import AppLayouts.Communities.popups 1.0
+import AppLayouts.Communities.helpers 1.0
+
+SplitView {
+    Logs { id: logs }
+
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        Item {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+
+            PopupBackground {
+                anchors.fill: parent
+            }
+
+            Button {
+                anchors.centerIn: parent
+                text: "Reopen"
+
+                onClicked: dialog.open()
+            }
+
+            TransferOwnershipAlertPopup {
+                id: dialog
+
+                anchors.centerIn: parent
```

Comment:
Usually it's helpful to add stuff like that:
```
                closePolicy: Popup.NoAutoClose
                visible: true
                modal: false
```

Reply:
Okk! I'll update it in my next pr! Thanks for the suggestion!